pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
@jreback. I must be missing something here. If the only customized value in the docstring is the class, and that value is identical is both places it is output, why does the docstring need to be shared at all? Why not simply hardcode the value into a traditional docstring?